versions/run.gaeac6.ver

```diff
@@ -4,5 +4,11 @@ export spack_env=gsi-addon
 
 export perl_ver=5.38.2
 
+export hsi_mod_path="/usw/hpss/modulefiles"
+export hsi_ver=9.3
+
 source "${HOMEgfs:-}/versions/spack.ver"
+
+# Gaea uses a newer version of Fit2Obs
+export fit2obs_ver=1.1.5
```

Comment:
@DavidHuber-NOAA Please also add `export ens_tracker_ver=v1.2.0` to this ver file. We can update the other platforms via this variable in `spack.ver` once I install and test this new version everywhere else. Thanks!

Reply:
Done and tested.